Bottom: d07098dd71542b3a6c06fd4823e133dd03d69228
Top:    14efe750c226d35f6d72868c7fe2f22a86225bf0
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-18 22:42:36 +0100

Refresh of give-the-hids-control-over-att

---

diff --git a/src/crosshair.c b/src/crosshair.c
index fb46c66..62bde1c 100644
--- a/src/crosshair.c
+++ b/src/crosshair.c
@@ -757,6 +757,26 @@ CrosshairOff (void)
     notify_mark_change (true);
 }
 
+/* ---------------------------------------------------------------------------
+ * Convenience for plugins using the old API
+ */
+void
+HideCrosshair (void)
+{
+  notify_crosshair_change (false);
+  notify_mark_change (false);
+}
+
+void
+RestoreCrosshair (void)
+{
+  notify_crosshair_change (true);
+  notify_mark_change (true);
+}
+
+/* ---------------------------------------------------------------------------
+ * Returns the square of the given number
+ */
 static double
 square (double x)
 {
diff --git a/src/crosshair.h b/src/crosshair.h
index 44b2c81..94ae1f6 100644
--- a/src/crosshair.h
+++ b/src/crosshair.h
@@ -46,12 +46,12 @@
 #define	STATE_SECOND	1
 #define	STATE_THIRD		2
 
+void notify_crosshair_change (bool changes_complete);
+void notify_mark_change (bool changes_complete);
 void CrosshairOn (void);
 void CrosshairOff (void);
 void DrawAttached (void);
 void DrawMark (void);
-void notify_crosshair_change (bool changes_complete);
-void notify_mark_change (bool changes_complete);
 void MoveCrosshairRelative (LocationType, LocationType);
 bool MoveCrosshairAbsolute (LocationType, LocationType);
 void SetCrosshairRange (LocationType, LocationType, LocationType,
